dgraphtest/cluster.go

```diff
@@ -161,6 +211,181 @@ func (hc *HTTPClient) RunGraphqlQuery(params GraphQLParams, admin bool) ([]byte,
 	return gqlResp.Data, nil
 }
 
+// LoginParams stores the information needed to perform a login request.
+type LoginParams struct {
+	Endpoint   string
+	UserID     string
+	Passwd     string
+	Namespace  uint64
+	RefreshJwt string
+}
+
+type GraphQLResponse struct {
+	Data       json.RawMessage        `json:"data,omitempty"`
+	Errors     x.GqlErrorList         `json:"errors,omitempty"`
+	Extensions map[string]interface{} `json:"extensions,omitempty"`
+}
+
+// HttpLogin sends a HTTP request to the server
```

Comment:
we shouldn't need this either, we already have LoginIntoNamespace & LoginUsingToken